tutorials/hello-kubernetes/README.md

```diff
@@ -20,11 +20,116 @@ cd quickstarts/tutorials/hello-kubernetes
+✅  Success! Dapr has been installed to namespace dapr-system. To verify, run `dapr status -k' in your terminal. To get started, go here: https://aka.ms/dapr-getting-started
+```
+
+### Step 2 - Run the Multi-app run template to deploy both the Node.js and Python apps
+
+To run both the Node.js and Python apps, run the following command from the `hello-kubernetes` directory:
+<!-- STEP
+name: "Run hello-kubernetes multi-app run template"
+tags:
+  - multi-app
+timeout_seconds: 60
+expected_return_code:
+expected_stdout_lines:
+  - 'Deploying app "nodeapp" to Kubernetes'
+  - 'Deploying app "pythonapp" to Kubernetes'
+  - '== APP - nodeapp == Got a new order! Order ID: 12'
+  - '== APP - nodeapp == Successfully persisted state for Order ID: 12'
+expected_stderr_lines:
+output_match_mode: substring
+match_order: none
+-->
+```bash
+dapr run -k -f dapr.yaml
+```
+
+<!-- END_STEP -->
+
+Expected output
+```
+ℹ️  This is a preview feature and subject to change in future releases.
+ℹ️  Validating config and starting app "nodeapp"
+ℹ️  Deploying app "nodeapp" to Kubernetes
+ℹ️  Deploying service YAML "/path/quickstarts/tutorials/hello-kubernetes/node/.dapr/deploy/service.yaml" to Kubernetes
+ℹ️  Deploying deployment YAML "/path/quickstarts/tutorials/hello-kubernetes/node/.dapr/deploy/deployment.yaml" to Kubernetes
+ℹ️  Streaming logs for containers in pod "nodeapp-6dcddb44f5-q5gnr"
+ℹ️  Writing log files to directory : /path/quickstarts/tutorials/hello-kubernetes/node/.dapr/logs
+ℹ️  Validating config and starting app "pythonapp"
+ℹ️  Deploying app "pythonapp" to Kubernetes
+ℹ️  Deploying deployment YAML "/path/quickstarts/tutorials/hello-kubernetes/python/.dapr/deploy/deployment.yaml" to Kubernetes
+== APP - nodeapp == Node App listening on port 3000!
+ℹ️  Streaming logs for containers in pod "pythonapp-7479cdcb7b-z827w"
+ℹ️  Writing log files to directory : /path/quickstarts/tutorials/hello-kubernetes/python/.dapr/logs
+ℹ️  Starting to monitor Kubernetes pods for deletion.
+== APP - nodeapp == Got a new order! Order ID: 2
+== APP - nodeapp == Successfully persisted state for Order ID: 2
+== APP - nodeapp == Got a new order! Order ID: 3
+== APP - nodeapp == Successfully persisted state for Order ID: 3
+== APP - nodeapp == Got a new order! Order ID: 4
+== APP - nodeapp == Successfully persisted state for Order ID: 4
+== APP - nodeapp == Got a new order! Order ID: 5
+== APP - nodeapp == Successfully persisted state for Order ID: 5
+```
+
+Use `Ctrl+C` to stop the apps. Or you can run the following command to stop the apps:
+<!-- STEP
+name: "Stop hello-kubernetes multi-app run template"
+tags:
+  - multi-app
+timeout_seconds: 60
+expected_return_code:
+expected_stdout_lines:
+expected_stderr_lines:
+-->
+<!-- Properly stop multi-app run template if not already stopped.-->
+
+```bash
+dapr stop -k -f dapr.yaml
+```
+
+<!-- END_STEP -->
+
+This spins down the Kubernetes resources that were deployed in the previous step.
+
+## Using the `kubectl` CLI
```

Comment:
Add a tab here, to make this tutorial shorter. Can then either use Multi-app run or kubectl

Reply:
Adding tabs is possible in Hugo which we use for docs, is it possible in normal Github Readme?

Reply:
No, it's not something we can do in GH